ractor_actors/src/net/tcp/session.rs

```diff
@@ -188,6 +206,7 @@ where
                 } else {
                     tracing::error!("TCP Session received a child panic from an unknown child actor ({}) - '{}'", actor.get_id(), panic_msg);
                 }
+                myself.stop_children(Some("session_stop_panic".to_string()));
```

Comment:
this is done automatically on teardown, is there a reason you need to do this manually?

Reply:
Yes, at least I did when working on the other branch I wanted to merge.

I think the problem was that I had two actors that I wanted to be linked with each other, but it doesn't seem possible with linking that these are co-dependent.

In Erlang this would be solved with a third supervisor that used the "one for all" strategy. Not sure what the best pattern would be here.